continuum-auth/src/use-cases/index.test.ts

```diff
@@ -0,0 +1,18 @@
+import { Request, Response } from 'express';
+import { HealthCheck } from './index';
+
+jest.mock('../logger');
+
+describe('healthcheck', () => {
+    it.only('is OK', () => {
```

Comment:
.only here is redundant as there is only one test in the file.

Reply:
Removed. Thanks.